spring-statemachine-core/src/main/java/org/springframework/statemachine/config/AbstractStateMachineFactory.java

```diff
@@ -37,55 +28,32 @@
 import org.springframework.statemachine.access.StateMachineAccess;
 import org.springframework.statemachine.access.StateMachineFunction;
 import org.springframework.statemachine.action.Action;
-import org.springframework.statemachine.config.model.ChoiceData;
-import org.springframework.statemachine.config.model.EntryData;
-import org.springframework.statemachine.config.model.ExitData;
-import org.springframework.statemachine.config.model.HistoryData;
-import org.springframework.statemachine.config.model.JunctionData;
-import org.springframework.statemachine.config.model.StateData;
-import org.springframework.statemachine.config.model.StateMachineModel;
-import org.springframework.statemachine.config.model.TransitionData;
-import org.springframework.statemachine.config.model.TransitionsData;
+import org.springframework.statemachine.config.model.*;
```

Comment:
Don't add import wildcards(if you're using idea then I think it uses that automatically after some amount of imports from same package.).